### PR TITLE
Remove error message from toast notification

### PR DESCRIPTION
**Pull request by dweberdev**
Targeting issue #3 and #4 for ticket #393

### CODE REVIEW

File: context/TransactionContext/index.tsx [hunk 1]
```diff
@@ -180,10 +180,7 @@ export const TransactionStore: React.FC = ({ children }: Children) => {
                 });
             } else {
                 updateToast(toastId as unknown as string, {
-                    content: [
-                        statusMessages?.error?.title ?? 'Transaction failed',
-                        statusMessages?.error?.body ?? error.message,
-                    ],
+                    content: [statusMessages?.error?.title ?? 'Transaction failed'],
```
Comment (dweberdev) [inline context/TransactionContext/index.tsx:183]: Is there any reason we would want to display the error message body?

File: archetypes/Stake/StakeGeneric/index.tsx [hunk 2]
```diff
@@ -295,6 +295,7 @@ export default (({
                     },
                     error: {
                         title: `Claim TCR Failed`,
+                        body: '',
```
Comment (dospore) [inline archetypes/Stake/StakeGeneric/index.tsx:298]: if you removed the body in the transaction context you shouldnt need this

File: context/TransactionContext/index.tsx (hunk 1 above)
Comment (dweberdev) [reply context/TransactionContext/index.tsx:183]: Good point, should we make a case to keep it or go ahead and remove it?

**Comment by dweberdev** [comment]
Closing PR, toast notification is getting updated

**Pull request closed without merging**